README.md

```diff
@@ -0,0 +1,42 @@
+# ereolengo.dk
+
+Modules for ereolengo.dk.
+
+## Coding standards
+
+All code must adhere to [the Drupal Coding
+Standards](https://www.drupal.org/docs/develop/standards).
+
+Install tools to check coding standards:
+
+```sh
+docker run --rm -it --volume $(pwd):/app itkdev/php7.0-fpm:latest composer2 install
+```
+
+Check the coding standards:
+
+```sh
+docker run --rm -it --volume $(pwd):/app itkdev/php7.0-fpm:latest composer2 coding-standards-check
```

Comment:
Also use docker-compose and if not lose the `-it` you don't need interactive terminal. Same for the next command.

Reply:
Fixed.